cpp/test/IceSSL/configuration/AllTests.cpp

```diff
@@ -1467,6 +1467,8 @@ allTests(Test::TestHelper* helper, const string& /*testDir*/, bool p12)
             try
             {
                 server->ice_ping();
+                info = ICE_DYNAMIC_CAST(IceSSL::ConnectionInfo, server->ice_getCachedConnection()->getInfo());
```

Comment:
Could we add a similar in Java? Unlike C++/C#, verified should be "true" since the hostname check is disabled if VerifyPeer=0. Adding the test in Java with a comment to explain the difference would be good.